### DISCUSSION TITLE
Dev gone

### DISCUSSION THREAD

**Discussion by DeadSecTr** [z-huang/music#1240] [General]
Are dev gone? Or why no more update of new version? There are many annoying bugs in the great app...

Like every good app who has been developed...at the and will these abandoned...donno why

**Comment by UcraNerd** [comment]
> Are dev gone? Or why no more update of new version? There are many annoying bugs in the great app...
> 
> 
> Like every good app who has been developed...at the and will these abandoned...donno why

Read https://github.com/z-huang/InnerTune/discussions/1122

**Comment by DeadSecTr** [comment]
Yes...4 months ago

**Reply by UcraNerd** [reply]
If you want an alternative that's being updated constantly there's rimusic, for the development of innertune idk

**Reply by SiddhantRakh** [reply]
I am so sad about the development of Innertune being halted like this. I loved it's UI. If you want a good alternative. My best suggestion is this fork of ViMusic by  25huizengek1(very active). I personally feel RiMusic has becomes bloated and laggy with unnecessary features. So you can try the following alternative. 
https://github.com/25huizengek1/ViMusic

**Reply by SiddhantRakh** [reply]
Or you can try this fork of Innertune itself by  Malopieds which was updated to fix some of the annoying bugs like Songs and Artists page not loading, metadata display issues and other bugs like album name being shown as the artist name.
https://github.com/Malopieds/InnerTune
